### PR TITLE
Openvpn new R2W policy certificate-otp

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6112

### CODE REVIEW

**Comment by stephdl** [comment]
Screenshot (25)

**Comment by stephdl** [comment]
Screenshot (26)

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -186,6 +186,8 @@
     "configure_rw_server": "Configure RoadWarrior server",
     "username_password_certificate": "Username, Password and Certificate",
     "username_password": "Username and Password",
+    "username_otp_certificate": "Username, One time password and Certificate",
+    "username_otp_certificate_waning_all_users_must_install_an_otp_application": "This VPN policy enforces the security, each system-user must enable his OTP setting and install an OTP application on his mobile phone",
```
Comment (gsanchietti) [inline ui/public/i18n/language.json:190]:
```suggestion
    "username_otp_certificate_waning_all_users_must_install_an_otp_application": "This policy enforces the OTP usage: users will be required to enable 2FA before accessing the VPN.",
```

**Review by gsanchietti — APPROVED** [review]
Feel free to merge if suggested change is good for you.

**Comment by stephdl** [comment]
obsoleted by https://github.com/NethServer/nethserver-vpn-ui/pull/24

**Pull request closed without merging**